app/error-handling.js

```diff
@@ -0,0 +1,17 @@
+const PSQLError = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Invalid ID, Please use a number" });
```

Comment:
consider a more vague error message, like 'bad request' ... the less information revealed about your database, the better